src/settings/toml/manifest.rs

```diff
@@ -93,6 +94,13 @@ impl Manifest {
 
         check_for_duplicate_names(&manifest)?;
 
+        if manifest.compatibility_date.is_none() {
+            StdOut::warn(&format!(
+                "Your configuration file is missing compatibility_date, so a distant past date is assumed. To get the latest possibly-breaking bug fixes, add this line to your wrangler.toml:\n\n    compatibility_date = \"{}\"\n",
+                Utc::now().format("%F")));
+            StdOut::warn("For more information about compatibility dates, see: https://developers.cloudflare.com/workers/platform/compatibility-dates");
```

Comment:
This URL doesn't seem to exist yet? Is that being worked on in parallel?

Reply:
Sorry not Kenton but yes - https://github.com/cloudflare/cloudflare-docs/pull/2289